src/content/docs/kubernetes-pixie/kubernetes-integration/installation/upgrading-kubernetes.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Upgrading Kubernetes integration'
+tags:
+  - Integrations
+  - Kubernetes integration
+  - Upgrade
+metaDescription: "Learn how to upgrade the Kubernetes integration."
+---
+
+After a new version of Kubernetes has been released, you can upgrade the Kubernetes version running on control plane nodes and worker nodes in a cluster.
+
+## Upgrading our Kubernetes integration [#update]
+
+Our Kubernetes integration is under active development and we regularly release updates which include bug fixes, new features, and support for newer Kubernetes versions and cloud providers. We strongly recommend all our customers to regularly update the Kubernetes integration to get the best experience.
+
+### Upgrading using the guided install and Helm [#upgrade-helm]
+
+In order to update an installation that was deployed using Helm command provided by the [guided install](https://one.newrelic.com/nr1-core?state=51fbbd48-c8ca-ead9-bb90-af96e18d82a7), just go through the process and run the Helm command again. This will pull the new version of the chart and its dependencies and upgrade it to the latest version.
+
+If you want to upgrade manually your Kuberenetes instegration installed via Helm:
+
+1. Update the local chart repository:
+
+   ```shell
+   helm repo update
+   ```
+
+2. Update the release by running again the appropriate `helm upgrade --install ...` command in the section above:
+
+   ```shell
+   helm upgrade --install newrelic newrelic/nri-bundle \
+   --namespace newrelic --create-namespace \
+   -f values-newrelic.yaml
+   ```
+
+### Upgrading using the guided install and plain manifests [#update-manifests]
+
+<Callout variant="tip">
+   We encourage you to deploy our integration using Helm, as it provides a cleaner upgrade path compared to using manifests directly.
+</Callout>
+
+If custom manifests have been used instead of Helm, you will need to first remove the old installation using `kubectl delete -f previous-manifest-file.yml`, and then proceed through the guided install again. This will generate an updated set of manifests that can be deployed using `kubectl apply -f manifest-file.yml`.
+
+We don't recommend applying a new version of the manifest file without removing the previous one first, since it might leave some leftover components in your cluster.
+
+If you need an advanced way to create your manifest, template it using Helm.
+
+## Upgrading to the new version of KSM [#update-ksm]
```

Comment:
Most likely users don't know what is to upgrade "to the last version of KSM", as it is included as a dependency. I would rephrase this using the `nri-bundle` version (from v4 to v5).

Reply:
Modified, from _Upgrading to the new version of KSM_ to _Upgrading the nri-bundle version_.